lib/ansible/module_utils/urls.py

```diff
@@ -81,6 +81,16 @@
 # agrees to be bound by the terms and conditions of this License
 # Agreement.
 
+'''
+The **urls** utils module offers a replacement for the `requests`_ library for most cases. 
+
+Sometimes you really need the requests library, but you should always check, whether 
+this utils module is sufficient for your use case.
+
+.. _requests:
+   http://docs.python-requests.org/en/master/
+'''
```

Comment:
This module is really a replacement for ```urllib.openurl()``` which should never be used because it suffers from security limitation.  requests has similar security features as this module_util but is an external dependency and therefore would be an additional requirement on the remote machine.